use-cases/README.md

```diff
@@ -1,8 +1,6 @@
-# Secure for Cloud for AWS Use-Cases
+# Secure for Cloud Use Cases for AWS Environments
```

Comment:
this index is not only aws specific, but will address that in further PRs. let's close this one